### PR TITLE
Add config for release notes generation

### PR DESCRIPTION
**Pull request by sisou**
https://docs.github.com/en/repositories/releasing-projects-on-github/automatically-generated-release-notes#configuring-automatically-generated-release-notes

### CODE REVIEW

File: .github/release.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+changelog:
+  categories:
+    - title: Breaking Changes
```
Comment (jsdanielh) [inline .github/release.yml]:
```suggestion
    - title: Protocol Breaking Changes
```

File: .github/release.yml [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+      labels:
+        - testnet-restart
+    - title: Updates
```
Comment (jsdanielh) [inline .github/release.yml]: Maybe, _Other Changes_?

Comment (sisou) [reply .github/release.yml]: What if there a no protocol breaking changes? This is the default category, which in the future should be the main category once there are no more breaking changes.

Comment (jsdanielh) [reply .github/release.yml]: Yeah, not sure. To me _Updates_ sounds more like dependency updates but this is not that

Comment (sisou) [reply .github/release.yml]: The default without a config file is "What's changed". We can use that. The documentation suggests "Features" as the title.

Comment (jsdanielh) [reply .github/release.yml]: Yeah, I'll be ok with any of those 2

Comment (jsdanielh) [reply .github/release.yml]: @sisou Will change it to "What's changed" to merge this PR. wdyt?

Comment (sisou) [reply .github/release.yml]: Sure!

**Review by jsdanielh — APPROVED** [review]

**Pull request merged**